addons/stock/models/stock_quant.py

```diff
@@ -877,6 +877,10 @@ def _get_inventory_move_values(self, qty, location_id, location_dest_id, out=Fal
             name = _('Product Quantity Confirmed')
         else:
             name = _('Product Quantity Updated')
+
+        if self.inventory_date:
+            name += f' [Scheduled on {self.inventory_date}]'
```

Comment:
Ah sorry, I missed this earlier. You need to add a `_()` around this for the translation. Also its preferred to use %s over f'  (when only one variable to replace in a string): https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#use-translation-method-correctly

Same thing applies to  ```res_move['name'] += f' [Accounted on {force_period_date}]'``` in the stock_account module